### PR TITLE
mod changelog to incorporate JHU geocoding

### PR DESCRIPTION
**Pull request by jsharpna**
Geomapping refactoring of the indicators codebase have changed the meaning or calculation of some FIPS or State totals

* Fixes cmu-delphi/covidcast-indicators#217
* Fixes cmu-delphi/covidcast-indicators#236

### CODE REVIEW

**Comment by capnrefsmmat** [comment]
Is this also explained in the JHU signal page? It has a [list of coding exceptions](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/jhu-csse.html#geographical-exceptions), and I'm not sure if that's up-to-date.

**Comment by jsharpna** [comment]
> Is this also explained in the JHU signal page? It has a [list of coding exceptions](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/jhu-csse.html#geographical-exceptions), and I'm not sure if that's up-to-date.

It is not!  I'll propose modification to that as well then

**Comment by krivard** [comment]
I'd also recommend taking a look at the [overall geocoding page](https://github.com/jsharpna/delphi-epidata/blob/main/docs/api/covidcast_geography.md) to make sure it's still correct

**Comment by dshemetov** [comment]
Should we increment the "number of data revisions since ... " field? Not sure what constitutes a data revision.

**Comment by krivard** [comment]
> Should we increment the "number of data revisions since ... " field? Not
sure what constitutes a data revision.

No; that field pre-dates the data versioning system. At some point we
should overhaul all the documentation to remove it, but that's beyond the
scope of this PR.

**Comment by dshemetov** [comment]
@krivard the overall geocoding page looks good to me, though I'm not that familiar with the megacounty privacy mapping.

File: docs/api/covidcast-signals/jhu-csse.md [hunk 1]
```diff
@@ -59,36 +59,28 @@ should be interpreted purely as an artifact of data reporting and correction.
...
+### Puerto Rico
 
-|Borough Name       |County Name        |FIPS Code      |
-|-------------------|-------------------|---------------|
-|Manhattan          |New York County    |36061          |
-|The Bronx          |Bronx County       |36005          |
-|Brooklyn           |Kings County       |36047          |
-|Queens             |Queens County      |36081          |
-|Staten Island      |Richmond County    |36085          |
+JHU does not report Puerto Rico deaths at the municipal level, instead reporting them in the "Unassigned" category. We map this to megaFIPS 72000 and provide uses this information when reporting deaths at the state level.
```
Comment (krivard) [inline docs/api/covidcast-signals/jhu-csse.md:68]: > We map this to megaFIPS 72000 and provide uses this information when reporting deaths at the state level.

Did  you mean

> We map this to megaFIPS 72000 and provide uses _of_ this information when reporting deaths at the state level.

or 

> We map this to megaFIPS 72000 and _use_ this information when reporting deaths at the state level.

File: docs/api/covidcast_changelog.md [hunk 2]
```diff
@@ -44,6 +44,19 @@ Duplicate survey weights had corrupted historical figures for the following sign
...
+#### 1 September 2020
+
+NY Boroughs county FIPS (36005, 36047, 36061, 36081, 36085) are now split in proportion to the population of each county, instead of being reported in aggregate in FIPS 36061. 
+
+#### 30 September 2020
+
+The following changes were made to all `jhu-csse` signals related to geocoding
+- NY Boroughs county FIPS (36005, 36047, 36061, 36081, 36085) are now [differentiated by JHU](https://github.com/CSSEGISandData/COVID-19/issues/3084),
+- “Unassigned” and “Out of State” counts are assigned to a megaFIPS at the county level (XX000 where XX is the state FIPS code) and will now be incorporated into the state totals
+- The split of the Kansas City, Missouri into its four subcounties is now done with a more precise population proportion,
```
Comment (krivard) [inline docs/api/covidcast_changelog.md:56]: In `jhu-csse.md`, the edit says this split is done evenly, not according to a population proportion -- which is it?

File: docs/api/covidcast_changelog.md [hunk 3]
```diff
@@ -44,6 +44,19 @@ Duplicate survey weights had corrupted historical figures for the following sign
...
+- The split of the Dukes and Nantucket, Massachusetts FIPS codes into the two subcounties is now done with a more precise population proportion,
```
Comment (krivard) [inline docs/api/covidcast_changelog.md:57]: In `jhu-csse.md`, the edit says this split is done evenly, not according to a population proportion -- which is it?

**Review by krivard — CHANGES_REQUESTED** [review]
1 mangled sentence and two mutually conflicting statements to fix

File: docs/api/covidcast-signals/jhu-csse.md (hunk 1 above)
Comment (dshemetov) [reply docs/api/covidcast-signals/jhu-csse.md:68]: Thanks, it was the latter. Fixed.

File: docs/api/covidcast_changelog.md (hunk 2 above)
Comment (dshemetov) [reply docs/api/covidcast_changelog.md:56]: According to population, thanks.

**Review by krivard — APPROVED** [review]

File: docs/api/covidcast-signals/jhu-csse.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ grand_parent: COVIDcast API
 
 * **Source name:** `jhu-csse`
 * **Number of data revisions since 19 May 2020:** 1
-* **Date of last change:** [3 June 2020](../covidcast_changelog.md#jhu-csse)
+* **Date of last change:** [30 September 2020](../covidcast_changelog.md#jhu-csse)
```
Comment (krivard) [inline docs/api/covidcast-signals/jhu-csse.md:12]:
```suggestion
* **Date of last change:** [7 October 2020](../covidcast_changelog.md#jhu-csse)
```

File: docs/api/covidcast_changelog.md [hunk 5]
```diff
@@ -44,6 +44,19 @@ Duplicate survey weights had corrupted historical figures for the following sign
 
 We now include figures on Puerto Rico for all `jhu-csse` signals at the state level.
 
+#### 1 September 2020
+
+NY Boroughs county FIPS (36005, 36047, 36061, 36081, 36085) are now split in proportion to the population of each county, instead of being reported in aggregate in FIPS 36061. 
+
+#### 30 September 2020
```
Comment (krivard) [inline docs/api/covidcast_changelog.md:51]:
```suggestion
#### 7 October 2020
```

**Pull request merged**